src/table/components/TableWrapper.jsx

```diff
@@ -233,7 +145,7 @@ export default function TableWrapper(props) {
         sx={tableContainerStyle}
         tabIndex={-1}
         role="application"
-        data-testid="table-wrapper"
+        data-testid="table-container"
```

Comment:
why?

Reply:
because it is the tableContainer, not wrapper

Reply:
aha, right, it is on TableContainer not on Paper.